extensions/amp-analytics/0.1/vendors.js

```diff
@@ -1123,10 +1123,14 @@ export const ANALYTICS_CONFIG = /** @type {!JSONType} */ ({
 
   'snowplow': {
     'requests': {
-      'aaVersion': 'amp-0.1',
+      'aaVersion': 'amp-0.2',
       'basePrefix': 'https://${collectorHost}/i?url=${canonicalUrl}&page=${title}&' +
           'res=${screenWidth}x${screenHeight}&stm=${timestamp}&' +
-          'tz=${timezone}&aid=${appId}&p=web&tv=${aaVersion}',
+          'tz=${timezone}&aid=${appId}&p=web&tv=${aaVersion}&' +
+          'cd=${screenColorDepth}&cs=${documentCharset}&' +
+          'duid=${clientId}&' +
```

Comment:
clientId takes a required param as scope (cookie name). this right now throws exception on the analytics-vendors.amp.html example page. @ecoron could you fix that?

/cc @avimehta